src/InputWithLabel.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect, useRef } from 'react';
+
+const InputWithLabel = ({ listTitle, handleTitleChange, children }) => {
+    const inputRef = useRef(null);
+
+    useEffect(() => {
+        if (inputRef.current) {
+            inputRef.current.focus();
+        }
+    }, []);
+
+    return (
+        <label>
+            {children}
+            <input
+                ref={inputRef}
+                value={listTitle}
+                onChange={handleTitleChange}
+                name="title"
```

Comment:
These should also not be hard-coded to be "title" since we could make this input hold completely different data. Maybe we could pass in another prop called "id" and use that value for the `name` and `id` properties here?

Reply:
You may not need the "name" property here anymore if you are using an ID, since we would typically use that in order to select this particular element of the form and we got rid of that code a little while ago. I would try removing it and seeing if you get any console warnings (I might be wrong here).